docs/source/quick-start.mdx

```diff
@@ -62,26 +62,34 @@ Once you have your User Access Token, run the following command in your terminal
 huggingface-cli login
 ```
 
-Or if you prefer to work from a Jupyter or Colaboratory notebook, then use [`login`]:
+Alternatively, you can programmatically login using [`login`] in a notebook or a script:
 
 ```py
 >>> from huggingface_hub import login
 >>> login()
 ```
 
 <Tip>
-
-You can also provide your token to the functions and methods. This way you don't need to
-store your token anywhere.
-
+It is also possible to login programmatically without asking the user for the token by directly
+passing the token to [`login`] (i.e. `login(token="hf_xxx")`). If you do so, be careful when
+sharing your source code. It is a best practice to load the token from a secure vault instead
+of saving it in plain in your codebase/notebook.
 </Tip>
 
 <Tip>
+Another way to authenticate is to provide your token when calling every methods. This way your token
+is never stored on the machine.
+</Tip>
 
+<Tip>
```

Comment:
I think it'd be nice to also clarify here that:

Once you are logged in, all requests to the Hub - even methods that don't require authentication - will use your access token by default.